apps/google-analytics-4/lambda/src/services/google-api.ts

```diff
@@ -153,39 +157,44 @@ export class GoogleApi {
     return accountSummaries;
   }
 
-  async runReport(property: string, slug: string, startDate?: string, endDate?: string, dimensions?: string[], metrics?: string[]) {
+  async runReport(
+    property: string,
+    slug: string,
+    startDate?: string,
+    endDate?: string,
+    dimensions?: string[],
+    metrics?: string[]
+  ) {
     const ONE_WEEK = 7 * 24 * 60 * 60 * 1000;
-    const DEFAULT_DIMENSIONS = [
-      'date',
-    ]
-    const DEFAULT_METRICS = [
-      'screenPageViews',
-      'totalUsers',
-      'screenPageViewsPerUser',
-    ]
+    const DEFAULT_DIMENSIONS = ['date'];
+    const DEFAULT_METRICS = ['screenPageViews', 'totalUsers', 'screenPageViewsPerUser'];
 
     try {
       const [response] = await this.betaAnalyticsDataClient.runReport({
         property: property,
         dateRanges: [
           {
-            startDate: startDate ?? (new Date(Date.now() - ONE_WEEK)).toISOString().split('T')[0],
+            startDate: startDate ?? new Date(Date.now() - ONE_WEEK).toISOString().split('T')[0],
```

Comment:
Maybe put a comment here for other people and our future-selves to say that this formats to `yyyy-mm-dd`